### PR TITLE
[5G: MVC][component: agw][type: enhancement] Amf core dependent files

### PR DESCRIPTION
**Pull request by chandra-77**
## Summary

Containd AMF-CORE PR's related changes

## Test Plan
S1AP integ_tests are passed. attached the log
[S1AP_integ_tests_Results.log](https://github.com/magma/magma/files/6280456/S1AP_integ_tests_Results.log)

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ulaskozat — CHANGES_REQUESTED** [review]
you need to generate go files for proto.

**Review by ulaskozat — COMMENTED** [review]
Please fix the redeclaration of LOG_NGAP

**Comment by chandra-77** [comment]
> 
> 
> Please fix the redeclaration of LOG_NGAP

Fixed

File: lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h [hunk 1]
```diff
@@ -105,10 +178,9 @@ typedef struct PDU_Session_resource_setup_request_s {
   ran_ue_ngap_id_t
       ran_ue_ngap_id;  // This IE uniquely identifies the UE association over
                        // the NG interface within the NG-RAN node
-  Ngap_PagingPriority_ty RAN_Paging_Priority;
+  Ngap_PagingPriority_ty RAN_Paging_Priority; /*optional*/
```
Comment (ulaskozat) [inline lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h]: should not this be `Ngap_PagingPriority_t`?

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**